cypress/integration/display_faqs.js

```diff
@@ -46,6 +46,43 @@ describe('Display Static FAQ', () => {
       .should('have.length', 4);
   });
 
+  it('FAQ Article should have all the necessary data', () => {
+    cy
+      .get('.faq-article-perex')
+      .children()
+      .should('exist');
+
+    cy.get('.faq-article-text').should('exist');
+
+    cy.get('.FAQArticleFeedback').should('exist');
+  });
+
+  it('User can submit some feedback on a FAQ article', () => {
+    const faqFeedback = 'This is my feedback!';
+    cy
+      .get('.FAQArticleFeedback')
+      .find('button')
+      .click();
+
+    cy.get('[data-cy=faq-article-form]').should('exist');
+
+    cy
+      .get('[data-cy=faq-article-form]')
+      .find('textarea')
+      .type(faqFeedback)
+      .should('have.value', faqFeedback);
+
+    cy
+      .get('[data-cy=faq-article-form]')
```

Comment:
This actually submits form..
Im not sure if we can use fixtures with graphql?

Reply:
Great point, i'm sure we can get around this by intercepting the requests while the server is running locally? 
Not sure about when testing against production tho, will look into alternatives.

Reply:
I would comment this out for now and raise an issue linking to this obstacle. So we don't swamp the backend with feedback instances.